docs/related_projects.rst

```diff
@@ -16,8 +16,8 @@ the two projects (see :ref:`h5py_pytable_cmp`), as does the
 
 .. seealso::
 
-   `HDF Group's list of HDF5 tools
-   <https://portal.hdfgroup.org/display/HDF5/HDF5+Tools+by+Category>`_
+   `HDF Group's list of HDF5 tools (TODO link)
```

Comment:
I can still see the old version of this page, with a deprecation notice, at https://docs.hdfgroup.org/archive/support/products/hdf5_tools/toolsbycat.html . The notice says it 'will remain online', though presumably at some point it will break.

https://portal.hdfgroup.org/documentation/hdf5/latest/_view_tools_command.html looks roughly similar on the new site, though it only covers command-line tools, not GUIs. :shrug:

Reply:
> I can still see the old version of this page, with a deprecation notice, at https://docs.hdfgroup.org/archive/support/products/hdf5_tools/toolsbycat.html . The notice says it 'will remain online', though presumably at some point it will break.

https://support.hdfgroup.org/archive/support/products/hdf5_tools/toolsbycat.html This is the old support website, not updated in 7 years, and never will. Definitely not to be used for anything.

> https://portal.hdfgroup.org/documentation/hdf5/latest/_view_tools_command.html looks roughly similar on the new site, though it only covers command-line tools, not GUIs. 🤷

https://support.hdfgroup.org/documentation/hdf5/latest/_view_tools_command.html This one is part of the libhdf5 documentation and I don't know if it will ever include third-part tools. That's why I did not want to use it.

Reply:
Fair enough, in that case let's just remove this seealso entirely.